egress_proxy/main.tf

```diff
@@ -61,31 +52,12 @@ resource "cloudfoundry_app" "egress_app" {
   }
 }
 
-###
-### Create a credential service for bound clients to use when make requests of the proxy
-###
 locals {
   https_proxy = "https://${random_uuid.username.result}:${random_password.password.result}@${local.egress_route}:61443"
   http_proxy  = "http://${random_uuid.username.result}:${random_password.password.result}@${local.egress_route}:8080"
   domain      = local.egress_route
   username    = random_uuid.username.result
   password    = random_password.password.result
-  protocol    = "https"
-  port        = 61443
-}
-
-resource "cloudfoundry_service_instance" "credentials" {
```

Comment:
Since we're no longer creating the UPSI, I think we should make it as easy as possible for callers to create the UPSI (and maybe show them how in the `README.md`). To that end, should we add the `jsonencode()`'d block below to a new output, eg `json_credentials`?